<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>478</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>374</width>
    <height>478</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>374</width>
    <height>478</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fritz VBox Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/res.qrc">
    <normaloff>:/gfx/icon.png</normaloff>:/gfx/icon.png</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QWidget" name="actionWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Anrufbeantworter</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Aktiviert:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLabel" name="enabledLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Ansage:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QComboBox" name="announcementBox"/>
          </item>
          <item row="4" column="0">
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="1">
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="bottomMargin">
              <number>12</number>
             </property>
             <item row="0" column="0">
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>ändern</string>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="default">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="4" column="2">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="newAnnounceBox">
         <property name="title">
          <string>Neue Ansage</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Es wurde eine neue Ansage gefunden!</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="newAnnounceName"/>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>speichern</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QWidget" name="progressWidget" native="true">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QProgressBar" name="progressBar">
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Fritz!Box wird kontaktiert...</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>374</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuExtras">
    <property name="title">
     <string>Extras</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuExtras"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEnable"/>
   <addaction name="actionDisable"/>
   <addaction name="separator"/>
   <addaction name="actionUpdate"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>Über</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Einstellungen</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionEnable">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/gfx/enable.png</normaloff>:/gfx/enable.png</iconset>
   </property>
   <property name="text">
    <string>aktivieren</string>
   </property>
   <property name="toolTip">
    <string>AB aktivieren</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDisable">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/gfx/disable.png</normaloff>:/gfx/disable.png</iconset>
   </property>
   <property name="text">
    <string>deaktivieren</string>
   </property>
   <property name="toolTip">
    <string>AB deaktivieren</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/gfx/update.png</normaloff>:/gfx/update.png</iconset>
   </property>
   <property name="text">
    <string>aktualisieren</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionEnable</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>enableVB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDisable</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>enableVB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showPreferences()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUpdate</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveNewAnnouncement()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>setAnnouncement()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enableVB()</slot>
  <slot>showAbout()</slot>
  <slot>showPreferences()</slot>
  <slot>updateAll()</slot>
  <slot>saveNewAnnouncement()</slot>
  <slot>setAnnouncement()</slot>
 </slots>
</ui>
